src/types/provider.ts

```diff
@@ -134,3 +135,8 @@ export type EstimateFeeAction =
     };
 
 export type EstimateFeeResponseBulk = Array<EstimateFeeResponse>;
+
+export type TransactionSimulationResponse = {
+  trace: TransactionTraceResponse;
+  fee_estimation: EstimateFeeResponse;
+};
```

Comment:
```suggestion
export interface TransactionSimulationResponse {
  trace: TransactionTraceResponse;
  fee_estimation: EstimateFeeResponse;
};
```

After this, in `src/types/account.ts`.. add following
```
export interface TransactionSimulation extends TransactionSimulationResponse {
  fee_estimation: EstimateFee;
}
```
Use this as return type for account.simulateTransaction